src/Composer/ReleaseVersion.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Acquia\headless_lightning\Composer;
+
+use Acquia\Lightning\Composer\ReleaseVersion as BaseReleaseVersion;
+use Composer\Script\Event;
+
+/**
+ * Updates the version number in Headless Lightning's component info files.
+ */
+class ReleaseVersion extends BaseReleaseVersion {
```

Comment:
We don't need this class.

execute() doesn't do anything but call the parent method, so it's not necessary; and updateMakeFile() acts on any build-lightning.make found in the profile directory. If none are found, it effectively does nothing. So we can remove this class, and our composer.json can just call the script that comes with Lightning itself.